### DISCUSSION TITLE
Support Float64Array as SimpleGeometry flatCoordinates?

### DISCUSSION THREAD

**Discussion by bjornharrtell** [openlayers/openlayers#15096] [Ideas]
That would allow directly consume and render data as recieved in binary form if supported by the data format fx. FlatGeobuf and should be more performant and memory efficient than having to transform into Array first.

**Comment by ahocevar** [comment]
Thanks for the suggestion, @bjornharrtell. And good timing! @tschaub and I are planning to look into efficient internal data formats for parsing in workers and transferring to the main thread or the GPU next week at the [Munich code sprint](https://github.com/openlayers/openlayers/wiki/Munich-Sprint,-Sept-2023).

**Reply by bjornharrtell** [reply]
Very cool and I'm very interested what will come out of it! I hope to be able to attend in a future sprint. 🙂

**Reply by StefanBrand** [reply]
@ahocevar Is there any new information on the support of FlatGeobuf?

**Reply by ahocevar** [reply]
@StefanBrand Not yet, unfortunately.
